src/store/index.ts

```diff
@@ -288,18 +287,46 @@ export const store = createStore({
             // finds an injector for an address
             const injector = await web3FromAddress(request.senderAddress);
 
-            // sign and send the transaction 
-            await apiPromise.tx.balances
-                .transfer(request.receiverAddress, request.amount)
-                .signAndSend(request.senderAddress, { signer: injector.signer }, (response) => {
-                    if (response.isFinalized) {
-                        context.commit("decrementBusyCounter");
-                        context.commit("setTransferTokenSuccessStatus", true);
+            apiPromise
+                .tx
+                .balances
+                .transfer(request.receiverAddress, request.amount).signAndSend(request.senderAddress, { signer: injector.signer }, ({ dispatchError, isFinalized }) => {
+                    if (dispatchError) {
+                        if (dispatchError.isModule) {
+                            const decoded = apiPromise.registry.findMetaError(dispatchError.asModule);
+                            const { docs, name, section } = decoded;
+
+                            if (`${section}.${name}: ${docs.join(" ")}` === "balances.InsufficientBalance: Balance too low to send value") {
+                                context.commit("decrementBusyCounter");
+                                context.commit("setTransferTokenError", "The transaction was unsuccessful: Inability to pay some fees");
+                            }
+                        } else {
+                            context.commit("setTransferTokenError", "The transaction was unsuccessful, please try again");
```

Comment:
don't you need to decrement busy counter here? Maybe we can decrementBuyCounter in finally?